tools/create_collection/create_collection.xml

```diff
@@ -60,26 +60,26 @@
 
 .. class:: warningmark
       
-*NOTE*: Be advised, to use the in workflows. And prioritize the use of the normal 'checkbox in dataset history' tool otherwise.
+*NOTE*: Be advised, to use this tool in workflows. And prioritize the use of the normal 'checkbox in dataset history' tool otherwise.
       
 .. class:: infomark
 
 **What it does**
 
-Creates a dataset collection.
+Creates a dataset collection of type list.
+
+.. class:: warningmark
+      
+      + The tool was designed to be used in a workflow. Prioritize the use of the checkbox in dataset history when creating collections when not in a workflow.
```

Comment:
@ccedmendoza can you improve the purpose of this tool, I find it hard to understand. I would put something like "This tool allows to programmatically creates a list (collection) of datasets in a Galaxy workflow"